GetNOTAM.py

```diff
@@ -1,20 +1,107 @@
-import requests
-import credentials
-import json 
-
-# To add parameters to seach we need to append to the url like example below
-# https://external-api.faa.gov/notamapi/v1/notams?icaoLocation=KLAX&domesticLocation=JFK&notamType=N&classification=INTL&effectiveStartDate=YYYY-MM-DDTHH%3AMM&effectiveEndDate=YYYY-MM-DDTHH%3AMM&featureType=RWY&locationLongitude=65&locationLatitude=30&locationRadius=10&lastUpdatedDate=10&sortBy=domesticLocation&sortOrder=Asc 
-
-url = 'https://external-api.faa.gov/notamapi/v1/notams'
-headers = {'client_id': credentials.clientID,'client_secret': credentials.clientSecret}
-
-req = requests.get(url, headers=headers)
-
-parsed_req = json.loads(req.text)
-
-print(f"{req.status_code}\n\n")
-print(f"{req.headers}\n\n")
-print(f"{req.text}\n\n")
-
-with open("TestData/TestNOTAM.json", 'w') as json_file:
-    json.dump(parsed_req, json_file)
+from tkinter.tix import INTEGER
+import requests
+import credentials
+import json
+import urllib.parse
+import os
+import ZuluConverter
+import AirportsLatLongConverter
+
+
+
+# Base URL for the NOTAM API
+
+
+def merge_two_dicts(x, y):

```

Comment:
is this function used anywhere?  if so I've missed it.  if not, why do we have it?  if we need it, can we write unit tests for it?

Reply:
The function isn't necessary. I just deleted it.